tests/group_functions/group_scan.h

```diff
@@ -513,163 +290,173 @@ class invoke_init_joint_scan_group_same_type {
   }
 };
 
-template <int D, typename T>
+template <int D, typename T, typename Group, bool with_init, typename U,
+          typename OpT>
 class scan_over_group_kernel;
 
+// FIXME: hipSYCL has wrong arguments order: init and op are interchanged
+#ifdef SYCL_CTS_COMPILING_WITH_HIPSYCL
+template <typename Group, typename V, typename T, typename BinaryOperation>
+auto inclusive_scan_over_group_impl(Group g, V x, BinaryOperation binary_op,
+                                    T init) {
+  return sycl::inclusive_scan_over_group(g, x, init, binary_op);
+}
+#else
+template <typename Group, typename V, typename T, typename BinaryOperation>
+auto inclusive_scan_over_group_impl(Group g, V x, BinaryOperation binary_op,
+                                    T init) {
+  return sycl::inclusive_scan_over_group(g, x, binary_op, init);
+}
+#endif
```

Comment:
CTS is supposed to validate that implementation conforms to the specification. We can't adjust to implementation bugs here. Please, keep only standard version and remove all non-standard code.

Reply:
Removed this workaround and disable scan_over_group tests with init for hipSYCL.
Kept workaround for known_identity for hipSYCL, because it's not what is tested here and it's not implemented yet.

Reply:
> Kept workaround for known_identity for hipSYCL, because it's not what is tested here and it's not implemented yet.

I suggest we either use `known_identity` for all implementations or don't use it at all.

Reply:
Removed workaround for known_identity and disable tests for hipSYCL.